server/src/main/java/io/crate/expression/operator/any/AnyOperator.java

```diff
@@ -114,7 +117,7 @@ private static void reg(Functions.Builder builder, String name, FunctionFactory
 
     abstract boolean matches(T probe, T candidate);
 
-    protected abstract Query refMatchesAnyArrayLiteral(Function any, Reference probe, Literal<?> candidates, Context context);
+    protected abstract Query refMatchesAnyArrayLiteral(Function any, Reference probe, List<?> nonNullValues, Context context);
 
```

Comment:
Let's add `@NonNull` to the new List param - I guess it's non nullable as I didn't see null checks.

Reply:
Thanks, added